### DISCUSSION TITLE
How to customize calendar popover ?

### DISCUSSION THREAD

**Discussion by Enthouane** [ToolJet/ToolJet#10670] [Q&A]
Hi ! It's all in the title : i could not find in the documentation, in github issues nor in tutorials how to customize the contents of a calendar event popover. I can toggle its visibility but not mess with its content...

**Comment by PriteshKiri** [comment]
Hey @Enthouane,

Could you please elaborate on the use case you're working on? Just so you know, you can drag and drop components in the popover to customize it.
